CONFIGURATION.md

```diff
@@ -72,6 +72,9 @@ ssl_ca                                   |  *  |                 |
 ssl.crl.location                         |  *  |                 |               | low        | Path to CRL for verifying broker's certificate validity. <br>*Type: string*
 ssl.keystore.location                    |  *  |                 |               | low        | Path to client's keystore (PKCS#12) used for authentication. <br>*Type: string*
 ssl.keystore.password                    |  *  |                 |               | low        | Client's keystore (PKCS#12) password. <br>*Type: string*
+ssl.engine.location                      |  *  |                 |               | low        | Path to OpenSSL engine library. OpenSSL >= 1.1.0 required. <br>*Type: string*
+ssl.engine.id                            |  *  |                 |               | low        | OpenSSL engine id is the name used for loading engine. Default value is "dynamic". <br>*Type: string*
+ssl.engine.callback_data                 |  *  |                 |               | low        | OpenSSL engine callback data (set with rd_kafka_conf_set_engine_callback_data()). <br>*Type: pointer*
```

Comment:
Use all-underscore names for programmatical properties.  ssl_engine_callback_data